### PR TITLE
Design fixes for Documentation page

### PR DESCRIPTION
**Pull request by anishaswain**
Fixes:
- On hovers, need to turn to the light teal colour (#1BC1C9), not the dark 
- Several instances of Overpass font converted to Work Sans
- There was too much space around the cards (they were not left-aligned with the rest of the page content), and the three cards do NOT need to be sitting on an additional shadow card.
- The cards should all auto-adjust to be the height of the tallest card, making them all the same height 
- The three recent meetings cards were too far apart, so switched the spacing to `margin: var( --pf-global--spacer--md);`
- The `quicklinks` should be left-aligned. 
- The dropdowns for the cards (Pbench agent, Pbench server, and Dashboard) should NOT use all caps -- use sentence case.
- There was a bit of an issue with the interaction of the dropdown on the cards as well. When the card is expanded, the icon would be down, and it needs to swap to pointing to the right on closing.

### CODE REVIEW

**Review by portante — APPROVED** [review]

**Comment by portante** [comment]
@anishaswain, can you rebase this PR?

**Review by portante — CHANGES_REQUESTED** [review]
While this looks ready to go, it needs a rebase.  Thanks!

**Review by npalaska — APPROVED** [review]
Needs squashing I guess

File: index.css [hunk 1]
```diff
@@ -103,6 +102,11 @@ li.PbenchQueItem::after {
    background-image: url(./images/WebServer.png);
  }
 
+ .div4{
```
Comment (dbutenhof) [inline index.css:105]: Doesn't "div4" seem like a fairly generic name for a very specific background pattern?

**Comment by anishaswain** [comment]
closing in reference to #1917

**Pull request closed without merging**